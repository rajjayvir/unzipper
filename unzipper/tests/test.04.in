0x06
0x64 0x10 0x00
0x63 0x28 0x00
0x62 0x3c 0x00
0x61 0x4e 0x00
0x0a 0x5f 0x00
0x78 0x5f 0x80
0x00 0x00 0x00 0x00
