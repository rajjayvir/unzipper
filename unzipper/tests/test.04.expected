0x64 : 1 0
0x63 : 2 10
0x62 : 3 110
0x61 : 4 1110
0x0a : 5 11110
0x78 : 5 11111
